Forms/Schema/FormSchema.php

```diff
@@ -65,7 +69,8 @@ public function getState(Form $form) {
 
 		if($form->Message()) {
 			$state['messages'][] = [
-				'value' => $form->Message(),
+				// TODO Make form / field messages not always stored as html
+				'value' => ['html' => $form->Message()],
```

Comment:
I suggest same as above; Nullify if not set.

Also we agreed to change `messages` to a single `message`. Shall we change it for fields as well to simplify things and keep consistency?